packages/story-editor/src/app/story/utils/getStoryPropsToSave.js

```diff
@@ -43,12 +46,23 @@ function getStoryPropsToSave({ story, pages, metadata, flags }) {
     'backgroundAudio',
     'terms',
   ]);
-  const products = getAllProducts(pages);
-  const content = getStoryMarkup(story, pages, metadata, flags);
+
+  // pull story fonts from pages / elements
+  const cleandFonts = getStoryFontsFromPages(pages);
+  const cleanedPages = cleanElementFontProperties(pages);
```

Comment:
I think I asked it in the previous version already but are the fonts still added per element that they need to be cleaned every time when saving the story? Shouldn't migration take care of that?

Reply:
With this PR the only time story fonts gets updated is when saving (autosave, publish, preview etc...) --- there's no longer updates when fonts get inserted updated etc... 

So any "new" data written to the DB should be clean.

Not sure on the migration end.